spec/services/catalog/update_order_item_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe Catalog::UpdateOrderItem do
+  describe "#process" do
+    let(:request) do
+      { :headers => { 'x-rh-identity' => encoded_user_hash }, :original_url => 'whatever' }
```

Comment:
@eclarizio I think this is better https://github.com/ManageIQ/catalog-api/blob/d9646de9e48c7747e3e8765241a9bb9e57101d65/spec/support/service_spec_helper.rb#L15
we can add the original_url and x-rh-insights-request-id in that spec helper for everyone to use